core/src/main/scala/libretto/CoreLib.scala

```diff
@@ -204,11 +203,10 @@ class CoreLib[DSL <: CoreDSL](val dsl: DSL) { lib =>
   }
 
   object ContraExternalizer {
-    implicit def inportInstance[C]: ContraExternalizer[[x] =>> x -⚬ C] =
-      new ContraExternalizer[[x] =>> x -⚬ C] {
-        def lift[A, B](f: A -⚬ B): (B -⚬ C) => (A -⚬ C) =
-          f > _
-      }
+    given inportTo[C]: ContraExternalizer[[x] =>> x -⚬ C] with {
```

Comment:
Not sure about the name. Usually I would prefer anonymous givens. But in this case this seems to read better. WDYT?

Reply:
Or just `inport`.